content/community.html

```diff
@@ -17,6 +17,14 @@
 <h3>Getting Involved</h3>
 <div class="communityblock">
 <img src="/images/development.svg" alt="Development" title="Development" />
-<p>We are a small team, so anything you can do to help is greatly appreciated.<br /> See the <a href="http://wiki.openra.net/#development-topics" alt="development topics">development topics</a> on our wiki for more information.</p>
-<p>You can share custom content such as maps on the <a href="http://resource.openra.net/">OpenRA Resource Center</a> site.</p>
+<p>We are a small team, so anything you can do to help is greatly appreciated.</p>
+<p>See the <a href="http://wiki.openra.net/#development-topics" alt="development topics">development topics</a> on our wiki for more information.</p>
+</div>
+
+<hr />
+<h3>Getting Content</h3>
+<div class="communityblock">
+<img src="/images/shared.svg" alt="Sharing" title="Sharing" />
+<p>You can share custom content such as maps on the <a href="http://resource.openra.net/">OpenRA Resource Center</a> site.<br />
+Replays, strategies and videos can be published at <a href="http://www.gamereplays.org/openra/">GameReplays.org</a> OpenRA section.</p>
```

Comment:
either "at *the* GameReplays.org OpenRA section" or "at GameReplays.org's OpenRA section"